models/uniswap/base/uniswap_v3_base_trades.sql

```diff
@@ -0,0 +1,108 @@
+{{ config(tags=['dunesql'],
+    schema = 'uniswap_v3_base',
+    alias = alias('trades'),
+    partition_by = ['block_date'],
```

Comment:
We may add a column `block_month` similar to `block_date` and partition by `block_month`
```suggestion
    partition_by = ['block_month'],
```